server/config/config.go

```diff
@@ -739,6 +739,7 @@ func (c *Config) GenEmbedEtcdConfig() (*embed.Config, error) {
 	cfg.PeerTLSInfo.TrustedCAFile = c.Security.CAPath
 	cfg.PeerTLSInfo.CertFile = c.Security.CertPath
 	cfg.PeerTLSInfo.KeyFile = c.Security.KeyPath
+	//nolint:staticcheck
```

Comment:
why?

Reply:
after https://github.com/etcd-io/etcd/pull/18015, AllowedCN is Deprecated